app/controllers/home_controller.rb

```diff
@@ -0,0 +1,9 @@
+class HomeController < ApplicationController
+  def index
+    if params[:"Search Merchants"]
```

Comment:
a more minor comment, but I would expect this to either be `params["Search Merchants"]` or `params[:search_merchants]`.